odoo/addons/base/tests/test_translate.py

```diff
@@ -593,6 +593,27 @@ def test_field_selection(self):
         self.assertEqual(fg['state']['selection'],
                          [('manual', 'Custo'), ('base', 'Pas touche!')])
 
+class TestTranslationCopy(TransactionCase):
+    def setUp(self):
+        super().setUp()
+
+    # opw-2266864
+    def test_01_copy_from_fr_to_en_with_no_en_name (self):
+        self.env['res.lang'].load_lang('fr_FR')
+        self.env['res.users'].with_context(active_test=False).search([]).write({'lang': 'fr_FR'})
+
+        langs = self.env['res.lang'].get_installed()
+        self.assertEqual([('en_US', 'English (US)'), ('fr_FR', 'French / Français')], langs, "Test did not started with expected languages")
+
+        partner = self.env['res.partner'].with_context(lang='fr_FR').create({'name': 'French Name'})
```

Comment:
but the `name` is not translatable on `res.partner`. Did you mean `res.partner.category` ?